infra/gcp/ensure-test-storage.sh

```diff
@@ -0,0 +1,27 @@
+set -o errexit
```

Comment:
Please add the shebang for bash, the license header and a line or two about what the script does